ci_build/python/run_android_emulator.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
```

Comment:
FYI, this (and the scripts it depends on) was mostly copied from here:
https://github.com/microsoft/onnxruntime/blob/0888c6cc5926ac5debf85b105264933784591b82/tools/python/run_android_emulator.py